datamade/how-to
# Updates new-django-app template, connects #227

## Overview

This PR implements the changes enumerated in #227:

- Disables indexing by default
- Fixes bug in error template block
- Prepopulates several meta tags

## Testing Instructions

* Follow the instructions in the templates directory to build a new Django app from the `new-django-app` template.
* Run your new application and confirm you can view it in your browser without error
* View the index page source and confirm:
  * You see `<meta name="robots" content="noindex">` near the top.
  * The following meta tags are correctly prepopulated: og:title, og:url, og:image, twitter:title, twitter:url, twitter:image
* In your `docker-compose.yml` file, add the following environment variable to your app service:
  ```yml
  DJANGO_ALLOW_SEARCH_INDEXING: "True"
  ``` 
* `docker-compose down`, then `docker-compose up` to restart your app with the new environment variable value.
* View the page source again and confirm you see  `<meta name="robots" content="all">` near the top.
